config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   resources :startup, only: [:index]
   root to: 'startup#index'
-  resources :trucks
+  resources :trucks, :drivers
```

Comment:
do them each as separate resources please, as we want to be able to freely modify our resources when needed. also makes the file easier to read when they are individual resources